### DISCUSSION TITLE
Missions & Radiation

### DISCUSSION THREAD

**Discussion by ElementShadow** [FakeFishGames/Barotrauma#7806] [Ideas and suggestions]
Hello devs, newbie here. Just got this beautiful game from my friends and its alot of fun, but there some mechanics i cant understand so will type them here as problem and ideas.

![baro](https://user-images.githubusercontent.com/73000483/146412198-7ff107d7-c1cf-483a-9f22-9cca7e9e8037.jpg)

I made picture with game situation example, so how you can see we get on base 4 missions, some of them lead to right direction, but some of them lead into radiation with more missions, and i cant understand why its made like this.
Same going with game settings where you can put maximum round missions 10, when on bases you can take 4 (didnt saw more)

so what i suggesting its to change how missions work, for example change this settings to "how many missions you can get on bases 4min 10max" 
and just making that each node not in radiation can have maximum 4 missions (+beacon / extra non base missions)
this will fix that missions appear in  dead sectors, and have more in proper direction where you want to go from rads.

I never was in radiation and dont know what will happen, but i think its strange to get missions in sector where everything dead.
Also would be cool to have border to see where next radiation will be, like in FTL

I hope you will understand what i mean, english not my language :(

**Comment by ElementShadow** [comment]
Also forgot to mention, first starting point in my opinion not should have missions with 3 skulls or higher, bcs newbies will die to pirates and moloch

**Comment by MaaiG** [comment]
>but there some mechanics i cant understand...

[You're not alone...))](https://github.com/Regalis11/Barotrauma/discussions/7446)
